File: PrimeNumber.java
import java.util.Scanner;
class  PrimeNumber
{
	static boolean isPrime(int number){
		int factors = 0;
		for (int i = 1; i <= number; i++)
		{
			if(number % i == 0){
				factors++;
			}
		}
		if(factors == 2)
			return true;
		else
			return false;
	}
	public static void main(String[] args) 
	{
		Scanner scan = new Scanner(System.in);
		System.out.println("Enter a number:");
		int number = scan.nextInt();
		boolean result = isPrime(number);
		if(result)
			System.out.println("Given number " + number + " is a prime number");
		else
			System.out.println("Given number " + number + " is not a prime number");
	}
}

File: PrimeNumber.py
def is_prime(number):
    factors = 0
    for i in range(1,number+1):
        if number % i == 0:
            factors += 1
    if factors == 2:
        return True
    else:
        return False

number = int(input("Enter a number: "))
result = is_prime(number)
if result == True:
    print(f"Given number {number} is a prime number")
else:
    print(f"Given number {number} is not a prime number")
    
